tests/integ/test_dataframe.py

```diff
@@ -175,7 +175,7 @@ def test_read_stage_file_show(session, resources_path, local_testing_mode):
             session.read.option("purge", False)
             .schema(user_schema)
             .csv(test_file_on_stage)
```

Comment:
`_show_string` is an internal API. Therefore pass explicitly `_emit_ast`. We do not mark internal APIs with `@publicapi`.

Reply:
@sfc-gh-oplaton I tried to explain the reasoniing here.